### PR TITLE
fix: fiori 3 table sort col

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/2439

#### Please provide a brief summary of this pull request.
There are some code changes inside table component library.

Functionality with example is working fine, these are still documentation/tests missing.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-custom-columns-example.component.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<table fd-table [keys]="displayedColumns">
```
Comment (stefanoScalzo) [inline apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-custom-columns-example.component.html:1]: when hovering over one of the filter options and not the filter, it should use grab icon as cursor

**Comment by stefanoScalzo** [comment]
Getting error when filtering ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'ng-untouched: true'. Current value: 'ng-untouched: false'.

**Comment by stefanoScalzo** [comment]
After filtering, and deleting filtered text, the items do not come back

File: libs/core/src/lib/table/directives/table-cell.directive.spec.ts [hunk 2]
```diff
@@ -1,8 +0,0 @@
-import { TableCellDirective } from './table-cell.directive';
```
Comment (InnaAtanasova) [inline libs/core/src/lib/table/directives/table-cell.directive.spec.ts:1]: No tests for this directive?

File: libs/core/src/lib/table/directives/table-row.directive.spec.ts [hunk 3]
```diff
@@ -1,8 +1,128 @@
...
+
+    @ViewChild(TableRowDirective)
+    tableRow: TableRowDirective;
+
+    getElements(): HTMLCollection {
+        return document.getElementById('row').children;
+    }
+}
+
+fdescribe('TableRowDirective', () => {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/table/directives/table-row.directive.spec.ts:31]: forgot the `f`

Comment (JKMarkowski) [reply libs/core/src/lib/table/directives/table-row.directive.spec.ts:31]: That's why tests passed :D

File: apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-filter.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'tableFilter'
+})
+export class TableFilterPipe implements PipeTransform {
+    transform(values: any[], searchTerm: string, key: string): any[] {
+        console.log(searchTerm);
```
[the remark this answers is not in the document]
Comment (JKMarkowski) [reply apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-filter.pipe.ts:8]: done

File: apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-custom-columns-example.component.html (hunk 1 above)
Comment (JKMarkowski) [reply apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-custom-columns-example.component.html:1]: It's only example

**Comment by JKMarkowski** [comment]
@stefanoScalzo i think you checked wrong example. 
@InnaAtanasova tests has been fixed.

**Pull request merged**